communication/src/main/java/org/jboss/da/communication/aprox/impl/AproxConnectorImpl.java

```diff
@@ -0,0 +1,76 @@
+package org.jboss.da.communication.aprox.impl;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.List;
+
+import javax.ejb.Stateless;
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+
+import org.jboss.da.communication.aprox.api.AproxConnector;
+import org.jboss.da.communication.aprox.model.GA;
+import org.jboss.da.communication.aprox.model.GAV;
+import org.jboss.da.communication.aprox.model.GAVDependencyTree;
+import org.jboss.da.communication.aprox.model.VersionResponse;
+
+@Stateless
+public class AproxConnectorImpl implements AproxConnector {
+
+    public GAVDependencyTree getDependencyTreeOfRevision(String scmUrl, String revision,
+            String version) {
+        // TODO Auto-generated method stub
+        return null;
+    }
+
+    public GAVDependencyTree getDependencyTreeOfGAV(GAV gav) {
+        // TODO Auto-generated method stub
+        return null;
+    }
+
+    public List<String> getVersionsOfGA(GA ga) {
+        StringBuilder query = new StringBuilder("http://10.3.8.115/api/group/public/"); // update with appropriate aprox group
```

Comment:
Maybe the ip could be specified in the 'da-config.json' file in the common module?